### PR TITLE
Upgrades `io.netty` lib to 4.1.69.Final

### PR DESCRIPTION
**Pull request by regio**
Upgrades `io.netty` lib to 4.1.68.Final.

## Documentation

- [X] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ For information on changes in released versions of Teku, see the [releases page]
 ## Unreleased Changes
 
 ### Additions and Improvements
+- Upgraded `io.netty` lib to 4.1.68.Final, `okhttp` lib to 4.9.0, `io.vertex` lib to 3.9.9.
```
Comment (regio) [inline CHANGELOG.md]: It should be `io.netty` 4.1.69.Final, right?

Comment (ajsutton) [reply CHANGELOG.md]: Yep good spot, I forgot about the changelog entry.

File: gradle/versions.gradle [hunk 2]
```diff
@@ -23,8 +23,8 @@ dependencyManagement {
 
     dependency 'com.launchdarkly:okhttp-eventsource:2.3.2'
     
-    dependency 'com.squareup.okhttp3:okhttp:4.8.1'
-    dependency 'com.squareup.okhttp3:mockwebserver:4.8.1'
+    dependency 'com.squareup.okhttp3:okhttp:4.9.0'
+    dependency 'com.squareup.okhttp3:mockwebserver:4.9.0'
```
Comment (rolfyone) [inline gradle/versions.gradle]: 4.9.2 is out, do we need to only be on 4.9.0?

**Review by rolfyone — APPROVED** [review]
mostly looks good, probably want to change the changelog comment to reflect or remove netty version that we're actually using, and an outstanding question about 1 library version that's been since updated...

**Pull request merged**